Lib/idlelib/hyperparser.py

```diff
@@ -23,7 +23,7 @@
 
 
 class HyperParser:
-    def __init__(self, editwin, index):
+    def __init__(self, editwin, index, end_at_eol=True):
```

Comment:
Just a nit, but I prefer
```suggestion
    def __init__(self, editwin, index, *, end_at_eol=True):
```
for something like this.

Reply:
Great suggestion!

I'm still slowly chipping away at the deeply-ingrained "IDLE support very old Python versions" rule in my mind...

Reply:
One 3.7.8 is out, we can consider using new-in-3.8 features such as '/' in signatures and ':='.